etcdctl/ctlv3/command/del_command.go

```diff
@@ -44,6 +45,7 @@ func NewDelCommand() *cobra.Command {
 
 // delCommandFunc executes the "del" command.
 func delCommandFunc(cmd *cobra.Command, args []string) {
+	fmt.Fprintf(os.Stderr, "Warning: Data which keys match %v will be deleted.", delPrefix)
```

Comment:
1. Warning should be printed only when user is doing wide-range operation, i.e. either prefix delete or range delete, not a lookup-delete. 

2.  It's good when warning is 'constructive', i.e. user can fix the 'usage' to not see the warning. but for this we need expected flag for --range deletes.

Reply:
Thanks. You are right. 

> Warning should be printed only when user is doing wide-range operation

Fixes